Controllers/Mobile/v1/ProjectsController.cs

```diff
@@ -331,5 +332,46 @@ private async Task<IActionResult> UnLikeProjectAsync(ProjectLike storedProjectLi
             return Ok("Unliked successfully");
         }
 
+        [Authorize(Roles = $"{Constants.USER_TYPE_CLIENT},{Constants.USER_TYPE_FREELANCER}")]
+        [HttpPost("{projectId}/comments")]
+        public async Task<IActionResult> CreateCommentAsync([FromRoute] long projectId, [FromForm] CommentInputDTO commentInputDTO)
+        {
+            if (!ModelState.IsValid)
+                return CustomBadRequest();
+
+            User? authenticatedUser = await userManager.GetUserAsync(HttpContext.User);
+            if (authenticatedUser == null)
+                return Unauthorized(CreateErrorResponse(StatusCodes.Status401Unauthorized.ToString(),
+                    "Unauthorized user"));
```

Comment:
It automatically returns Unauthorized, so you don't need this.
Because It recognized JWT.